Gemfile

```diff
@@ -11,14 +11,16 @@ def location_for(place, fake_version = nil)
 end
 
 group :test do
-  gem "rspec-core",             '3.1.7',                  :require => false
-  gem "rspec-puppet",           '2.3.2',                  :require => false
+  gem "rspec-core",             '3.5.1',                  :require => false
+  gem "rspec-puppet",           '2.4.0',                  :require => false
```

Comment:
That does look more useful.
If the rest of the tests pass as is then it's good for me.